src/routes/(marketing)/(components)/ai.svelte

```
@@ -0,0 +1,227 @@
+                                    ? 'bg-[var(--primary-color)]'
+                                    : 'bg-white/40'}"
+                            ></div>
+                            {#if activeMobileIndex === i}
+                                <div
+                                    class="absolute inset-0 bg-gradient-to-tl from-(--primary-color)/4 to-(--secondary-color)/10"
+                                >
+                                    <Noise opacity={0.1} />
+                                </div>
+                            {/if}
+                        </button>
+                    {/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)